lib/cloudregister/VERSION

```diff
@@ -1 +1 @@
-10.1.8
+10.2.0
```

Comment:
DO not bump the version just yet, we know this feature is incomplete, we still need a differentiation between calling `get_activations()` for th eregistry path and `get_activations()` for the repository path such that we get different system credentials cache handling server side.

Reply:
Yes, Im not merging this PR yet
the extra param with whatever header name neded on the server side, will be added on this PR, but the version needs to be bumped anyway ? So when the PR is feature complete it can be merged and nothing else to do

Reply:
WE should not have this PR grow with an additional unknown number of commits. The basic structure is good, remove the version bump, fix the exit conditions in the function of the utils and we can merge the first step.